### DISCUSSION TITLE
Singularities should darken the players when approaching the Event Horizon

### DISCUSSION THREAD

**Discussion by Shibva** [HbmMods/Hbm-s-Nuclear-Tech-GIT#694] [Show and tell]
After asking a person who knows the mod well and asked for their opinion, they suggested I should coin this idea to improve the singularities through one aspect; mimicking the effect of one (players) going through or approaching the event horizon.

I've realized that the entity created from the Endest Pearl (item from Avaratia) has an interesting visual detail in regards to a player's proximity to the center (event horizon) in terms of one screen dimming until you get close enough where it goes black. This made me think of how it would be realistic considering light cannot escape past the Event Horizon and with that logic could cause such an effect as one grows closer to that point

I'll be demonstrating what I mean through visual representation from the item's effect itself

Before; kept my distance from the black sphere.
![image](https://user-images.githubusercontent.com/38770780/180636993-bf8c03a5-f2c5-4f66-91c7-94a5b152d04b.png)

here I grow closer to it; notice how the screen grows dimmer as I grew closer to it before it collapsed
![image](https://user-images.githubusercontent.com/38770780/180637034-62b88366-0bdb-436c-b0bd-9460e66beb4a.png)

From what Avaratia did, it appears that this effect (based on what methods they have done) does not seem to conflict with optifine (shaders do not count, naturally)

I considered sharing this and suggesting an improvement to the great visuals that the singularities currently have and consulted a good friend on their opinion that knows the mod well.

**Comment by username228a91** [comment]
it is not true the closer you are to blackhole the brighter it gets - accretion converts 50% of matter into energy and creates alot of energy and also light can get trapped near black hole and it is called photonic field as i remember
